### ISSUE TITLE
PeerDependency issue

### ISSUE THREAD

**Issue by xavier-dubruille** [Nolanus/ngx-page-scroll#429]
Hello hello,

ngx-page-scroll@7.0.4 has a peerDependency to ngx-page-scroll-core@7.0.3 instead of @7.0.4 !!

Beside not having your last code revision, it also lead to an issue when using current angular version (^11.0.0) and updating with current npm version (7):
because ngx-page-scroll@7.0.4 wants to install ngx-page-scroll-core@7.0.3 which is not compatible (as PeerDependecies) with angular ^11.0.0 ... and make the update fails :/

**Comment by xavier-dubruille** [comment]
One way to get around this, is to first update the 'core' package and then update the 'main one' with `--legacy-peer-deps`
```
npm update ngx-page-scroll-core
npm update --legacy-peer-deps ngx-page-scroll
```
Which will put both in version 7.0.4 :)

**Comment by Nolanus** [comment]
Sorry for the inconvenience, fixed with v7.0.5
